IaC/AKS/README.md

```diff
@@ -330,6 +332,49 @@ kubectl create secret generic ngsa-aks-secrets \
 
 ```
 
+## Setup SSL/DNS
+
+### DNS Setup
+
+Assuming that you have a domain name, and permissions to update DNS records for your domain, create an A record to point your domain to `INGRESS_PIP`.
+
+For example, in Azure:
+
+```bash
+
+az network dns record-set a add-record -g <dns zone resource group> -z <dns zone name> -n $Ngsa_Name -a $INGRESS_PIP --ttl <TTL seconds>
```

Comment:
can you provide an example of `dns zone resource group` and `dns zone name`? how can I obtain these? If am trying to set this up I wouldn't know where to get these records.